src/main/java/com/echobox/api/linkedin/util/ValidationUtils.java

```diff
@@ -60,5 +65,28 @@ public static void verifyParameterPresence(String parameterName, Object paramete
       throw new NullPointerException("The '" + parameterName + "' parameter cannot be null.");
     }
   }
-
+  
+  /**
+   * Validate response.
+   *
+   * @param response the response from the API call
+   */
+  public static void validateResponse(WebRequestor.Response response) {
+    // If there was no response error information and this was a 401
+    // error, something weird happened on LinkedIn's end. Assume it is a Oauth error.
+    if (HttpURLConnection.HTTP_UNAUTHORIZED == response.getStatusCode()) {
```

Comment:
Can we use `HttpStatus ` to replace `HttpURLConnection`? as `HttpStatus `has a larger set.